packages/moonstone/styles/colors.less

```diff
@@ -171,7 +171,8 @@
 
 // Notification
 // ---------------------------------------
-@moon-notification-border-color: fade(@moon-black, 30%);
+@moon-notification-border-color: #bbb;
```

Comment:
@viodragon2 Can you verify the same color should be used for the border in both light and dark skins? This is different than ContextualPopup.

Reply:
I was waiting for light skin doc for the final update, but I would assume the border color would be the same. I'll make an update.

Reply:
Let's revert this change if it wasn't specced for consistency.